nextflow.config

```diff
@@ -272,9 +267,18 @@ dag {
     enabled = true
     file    = "${params.outdir}/pipeline_info/pipeline_dag_${params.trace_report_suffix}.html"
 }
+prov {
+    enabled = true
+    formats {
+        bco {
+            file = "${params.outdir}/pipeline_info/manifest_${params.trace_report_suffix}.bco.json"
+        }
+    }
+}
 
 manifest {
     name            = 'nf-core/multiplesequencealign'
+    author          = """Luisa Santus, Jose Espinosa-Carrasco""" // The author field is deprecated from Nextflow version 24.10.0, use contributors instead
```

Comment:
Since your minimal NF version is > than 24.10.0 you can delete the `author` field here